inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/model/KnowledgeBase.java

```diff
@@ -137,6 +137,12 @@
     @Column(nullable = false)
     private IRI propertyDescriptionIri;
 
+    /**
+     * The IRI for a base concept for instance for DBPedia as owl:Thing
+     */
+    @Column(nullable = false)
```

Comment:
Uhm... we already have root concepts in the KB:

```
    /**
     * A List of explicitly defined root concepts that can be used if auto detection takes too long
     */
    @ElementCollection(fetch = FetchType.EAGER)
    @CollectionTable(name = "knowledgebase_root_classes")
    @Column(name = "name")
    private List<IRI> explicitlyDefinedRootConcepts = new ArrayList<>();
```

Why do we need this additional `baseConceptIri`?

Reply:
`explicitlyDefinedRootConcepts` is something which user can specify to limit the concept tree but its not something which can be taken as the base concept IRI. I gave a thought about it, `baseConceptIri` would be the IRI to define the base super class for ontologies and can be used further in the application as a separate variable.  Otherwise we would have to ask the user to change the `explicitlyDefinedRootConcepts` in case the default base concept is already defined and she decides to use some of the sub class concept. WDYS?

Reply:
The intention of introducing the `explicitlyDefinedRootConcepts` was to avoid having to run the auto-detection of root concepts. So if any `explicitlyDefinedRootConcepts` are defined, then `KBSI.listRootConcepts()` would return them instead of trying to auto-detect the root concepts. As far as I can see, it has been implemented in this way as well.

So why do we additionally need a `baseConceptIri`? 

From what I see in https://github.com/inception-project/inception/blob/6b2714e33f618dc553da42124a1b1808573a8e0e/inception-kb/src/main/java/de/tudarmstadt/ukp/inception/kb/KnowledgeBaseServiceImpl.java#L811 , I'd say that defining a `baseConceptIri` is equivalent to defining a single `explicitlyDefinedRootConcepts`. So I don't see why we would need another field in the KB and another if/else branch in the `KBSI.listRootConcepts()`.